letsencrypt/models/letsencrypt.py

```diff
@@ -149,7 +158,11 @@ def cron(self):
             crt.write(chain_cert.read())
             chain_cert.close()
             _logger.info('wrote %s', crt.name)
-        self.call_cmdline([
-            'sh', '-c', self.env['ir.config_parameter'].get_param(
-                'letsencrypt.reload_command', ''),
-        ])
+        reload_cmd = self.env['ir.config_parameter'].get_param(
+            'letsencrypt.reload_command', False)
+        if reload_cmd:
+            _logger.info('reloading webserver...')
+            self.call_cmdline(['sh', '-c', reload_cmd])
```

Comment:
no, we need some shell to allow multiple commands there. Sh shells are very wide-spread, so I don't see a problem there.

good idea about the group, done